### PR TITLE
fix: sdkmath type in PoolSnapshot

### PR DESCRIPTION
**Pull request by saiSunkari19**
## Description

Closes: #XXXX

typo fixes

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by MonikaCat** [comment]
Hey @saiSunkari19 can you also update go version to `1.18` to fix unit tests and include entry inside `CHANGELOG.md`? Thanks!

**Comment by saiSunkari19** [comment]
Sure, I will do that, but current upstream branch is with `1.19`. Do  you want me to update that

**Comment by MonikaCat** [comment]
Hi @saiSunkari19 yes please, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #### Staking Module
 - ([\#443](https://github.com/forbole/bdjuno/pull/443)) Remove tombstone status from staking module(already stored in slashing module)
 - ([\#455](https://github.com/forbole/bdjuno/pull/455)) Added `unbonding_tokens` and `staked_not_bonded_tokens` values to staking pool table
+- ([\#536](https://github.com/forbole/bdjuno/pull/536)  fix `PoolSnapshot` import from  `sdk.Int` to `sdkmath.Int`
```
Comment (MonikaCat) [inline CHANGELOG.md]:
```suggestion
- ([\#536](https://github.com/forbole/bdjuno/pull/536) Fix `PoolSnapshot` tokens type from  `sdk.Int` to `sdkmath.Int`
```

**Comment by MonikaCat** [comment]
Hey @saiSunkari19 can you update go version to 1.18 [on this line please](https://github.com/saiSunkari19/bdjuno/blob/saisunkari19/v0.46.x/.github/workflows/test.yml#L28)? Thanks 👍🏻

**Comment by saiSunkari19** [comment]
Updated, Thanks

**Review by MonikaCat — APPROVED** [review]

**Review by rmontagnin — APPROVED** [review]

**Pull request merged**